inference-engine/tests/functional/plugin/cpu/main.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "gtest/gtest.h"
+#include "functional_test_utils/layer_test_utils.hpp"
+
+int main(int argc, char* argv[]) {
+    FuncTestUtils::SkipTestsConfig::disable_tests_skipping = false;
+    for (int i = 0; i < argc; ++i) {
+        if (std::string(argv[i]) == "--disable_test_skips") {
```

Comment:
1. From the name `--disable_test_skips` I don't understand why option does.
2. Could we add this option to help when we run `./binaryName --gtest_help`?
3. It is possible to have a single file with main function and link with all plugin tests? I suppose `gtest_main` does this, we can do the same

Reply:
1. changed to `--disable_tests_skipping`  - a little bit more accurate formulation. Can't come up with something more clear
2. Added printing of an additional section with custom command-line options before main gtest help message
3. Need help here, Can't find a way how to add a single additional source file using `addIeTargetTest`

Reply:
Decided to implement 3rd item separately since it requires to update the common make function adding a targets